packages/dendron-cli/src/commands/exportPodV2.ts

```diff
@@ -0,0 +1,269 @@
+          dendronConfig: engine.config,
+        });
+      case PodV2Types.JSONExportV2:
+        return new JSONExportPodV2({
+          podConfig: config,
+        });
+      case PodV2Types.AirtableExportV2:
+        return new AirtableExportPodV2({
+          airtable: new Airtable({ apiKey: config.apiKey }),
+          config,
+          engine,
+        });
+      case PodV2Types.NotionExportV2:
+        return new NotionExportPodV2({
+          podConfig: config,
+        });
+      case PodV2Types.GoogleDocsExportV2: {
+        const { wsRoot } = engine;
+        const fpath = EngineUtils.getPortFilePathForCLI({ wsRoot });
+        const port = openPortFile({ fpath });
+
+        return new GoogleDocsExportPodV2({
+          podConfig: config,
+          engine,
+          port,
+        });
+      }
+
+      default:
+        throw new DendronError({
+          message: `the requested pod type :${config.podType} is not implemented yet`,
+        });
+    }
+  }
+
+  async execute(opts: CommandOpts): Promise<CommandOutput> {
+    const ctx = "execute";
+    const { server, serverSockets, engine, config, payload } = opts;
+    this.multiNoteExportCheck({
+      destination: config.destination,
+      exportScope: config.exportScope,
+    });
+    const pod = this.createPod(config, engine);
+    this.L.info({ ctx, msg: "running pod..." });
+    const exportReturnValue = await pod.exportNotes(payload);
+    await this.onExportComplete({
+      exportReturnValue,
+      podType: config.podType,
+      engine,
+      config,
+    });
+    this.L.info({ ctx, msg: "done execute" });
+    return new Promise((resolve) => {
+      server.close((err: any) => {
+        this.L.info({ ctx, msg: "closing server" });
+        // close outstanding connections
+        serverSockets?.forEach((socket) => socket.destroy());
+        if (err) {
+          return resolve({
+            error: new DendronError({ message: "error closing", payload: err }),
+          });
+        }
+        resolve({ error: undefined });
+      });
+    });
+  }
+
+  async onExportComplete(opts: {
+    exportReturnValue: any;
+    podType: PodV2Types;
+    engine: DEngineClient;
+    config: any;
+  }) {
+    const { exportReturnValue, podType, engine, config } = opts;
+    switch (podType) {
+      case PodV2Types.AirtableExportV2:
+        return this.onAirtableExportComplete({ exportReturnValue, engine });
+      case PodV2Types.GoogleDocsExportV2:
+        return this.onGoogleDocsExportComplete({ exportReturnValue, engine });
+      case PodV2Types.NotionExportV2:
+        return this.onNotionExportComplete({ exportReturnValue, engine });
+      case PodV2Types.MarkdownExportV2:
+        return this.onMarkdownExportComplete({ exportReturnValue, config });
+      case PodV2Types.JSONExportV2:
+        return this.onJSONExportComplete({ exportReturnValue, config });
+      default:
+        assertUnreachable(podType);
+    }
+  }
+
+  async onAirtableExportComplete(opts: {
```

Comment:
in the plugin, we print out the record id that is updated. we should do the same thing here

Reply:
this comment also applies for other pods